smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy

```diff
@@ -813,7 +813,7 @@ def deviceHandler(evt) {
 		}
 	} else if (deviceInfo) {
 		if (deviceInfo.callbackUrl) {
-			sendToHarmony(evt, deviceInfo.callbackUrl)
+			sendToHarmonyCloud(evt, deviceInfo.callbackUrl)
```

Comment:
It looks like this will work for the Harmony Cloud calls, but the code doesn't appear to be backwards compatible with existing ```deviceInfo.callbackUrl``` values, which may reference LAN IP addresses that will not work correctly for ```httpPostJson```. Since we are using the same SmartApp both for existing and new installations, you will probably need to inspect ```deviceInfo.callbackUrl``` to decide whether to call ```sendToHarmony``` or ```sendToHarmonyCloud```.

I'm not sure the best way to make that determination. Maybe you could parse it and if it contains an IP address, use ```sendToHarmony```, but if it is not an ip address, ```sendToHarmonyCloud```?